external_modules/total_energy_module/prepare_qe.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+while getopts q: flag
+do
+    case "${flag}" in
+        q) root_dir=${OPTARG};;
+    esac
+done
+
+cd $root_dir
+if [ "$(grep -q "$foxflags -fPIC" install/configure)" == "" ]; then
+  echo "HERE"
```

Comment:
Is this a "check" to make sure that the condition is met? If so then maybe make the script fail if grep doesn't find what it is supposed to find.

Reply:
This only checks if the foxflags command has already been added to the configure file. I think users may accidentally execute the script multiple times, so I added a check. If the lines are already there all is well and good :)

Reply:
I meant the `echo "HERE"` part which looks like a left over printf-debugging artifact.

Reply:
Ok. great!